### DISCUSSION TITLE
having issues setting up metagem environment

### DISCUSSION THREAD

**Discussion by ycsong** [franciscozorrilla/thesis_pipeline#140] [Q&A] [answered]
Hello,

I am running into some issue with setting up the metagem environment while having the mamba environment activated. I am doing this in MacOS (Ventura 13.6.1). 

```shell
(mamba) song102@WE49174 metaGEM % conda config --add channels conda-forge && mamba create --prefix envs/metagem -c bioconda metagem
Warning: 'conda-forge' already in 'channels' list, moving to the top

Looking for: ['metagem']

bioconda/osx-arm64                                            No change
bioconda/noarch                                               No change
conda-forge/osx-arm64                                9.0MB @   5.2MB/s  1.9s
conda-forge/noarch                                  14.8MB @   5.8MB/s  2.8s
Could not solve for environment specs
The following package could not be installed
└─ metagem   is uninstallable because it requires
   └─ samtools >=1.9 , which does not exist (perhaps a missing channel).
``` 

I tried installing samtools and exporting its path to the environment variables but to no avail. Is there a configuration file that I need to edit before setting this up?

Thank you in advance,

Young

**Comment by franciscozorrilla** [comment, marked as answer]
Hey Young, sorry I missed this discussion post. Indeed the conda metaGEM recipe was only tested on linux machines since the inteded use cases are for high performance comptuer clusters. However, others have requested this feature before so please have a look [at this post/issue](https://github.com/franciscozorrilla/metaGEM/issues/134#issuecomment-1645496792) for a temporary solution.

**Comment by ycsong** [comment]
Many thanks for your response, Francisco. Will give this a try.
